docs/docs/references/connectors/README.mdx

```diff
@@ -31,31 +31,42 @@ It's assigned by the developers of each connector and will be uploaded to DB.
 ### target
 
 _target_ is a lowercase string to distinguish the social identities source of the social connector.
-For email and SMS connectors, a _target_ works more like a human readable ID.
 
 For example, both _WeChat web connector_ and _WeChat native connector_'s _target_ are `wechat`. _Aliyun direct mail connector_'s _target_ is `aliyun-dm` and _Aliyun SMS connector_'s _target_ is `aliyun-sms`.
 
 ### type
 
+_type_ is the property that record the type of the connector.
+
 _type_ should be one of the following connector types:
```

Comment:
nit: We define the connectors into three different types, based on their functionalities.